mist-common/src/main/avro/task_to_master_msg.avpr

```diff
@@ -52,6 +52,10 @@
           "name": "TaskLoad",
           "type": "double"
         },
+        {
+          "name": "NumEventProcessors",
```

Comment:
Why do we need this information?

Reply:
Can't we send the group load info from Task by dividing the load by the number of event processors?